apps/www/components/ui/toggle-group.tsx

```diff
@@ -0,0 +1,85 @@
+"use client"
+
+import * as React from "react"
+import * as ToggleGroupPrimitive from "@radix-ui/react-toggle-group"
+import { VariantProps, cva } from "class-variance-authority"
+
+import { cn } from "@/lib/utils"
+
+const ToggleGroupContext = React.createContext<
+  VariantProps<typeof toggleGroupItemVariants>
+>({
+  size: "default",
+  variant: "default",
+})
+
+const toggleGroupItemVariants = cva(
+  "ml-[1px] inline-flex h-10 items-center justify-center transition-colors first:ml-0 first:rounded-l last:rounded-r text-sm font-medium dark:hover:bg-slate-800 data-[state=on]:bg-slate-200 dark:data-[state=on]:text-slate-100 dark:data-[state=on]:bg-slate-700 focus:outline-none focus:relative focus:ring-2 focus:ring-slate-400 focus:ring-offset-2 disabled:pointer-events-none disabled:opacity-50 dark:focus:ring-offset-slate-900 dark:text-slate-100 dark:hover:text-slate-100",
+  {
+    variants: {
+      variant: {
+        default:
+          "bg-slate-900 text-white  hover:bg-slate-700 dark:bg-slate-50 dark:text-slate-900 data-[state=on]:text-slate-900 ",
```

Comment:
some white space warning here and because of I like to see tailwind sorted className, I think this className isn't sorted with any rules at all.